packages/amplify_core/lib/amplify_core.dart

```diff
@@ -18,6 +18,7 @@ export 'src/category/amplify_categories.dart';
 
 /// Config
 export 'src/config/amplify_config.dart';
+export 'src/config/amplify_outputs/amplify_outputs.dart';
```

Comment:
Note: I think merge conflicts are causing changes to show in this PR that are not part of the PR. I am going to ignore files that I don't think are intended to be part of the PR and we re-review once the PR is updated.